sshh/main.py

```diff
@@ -110,10 +111,32 @@ def cmd_agent(request):
         sshenv['PS1'] = f"[{request.group}]{sshenv['PS1']}"
         logger.info('ssh-agent PID=%s session "%s" has been started. To close this session, exit shell.',
                     sshenv['SSH_AGENT_PID'], request.group)
-        subprocess.run (sshenv['SHELL'], env=sshenv)
+        shell_command = sshenv['SHELL']
+        bashrc_location = Path(sshenv.get('HOME', ''), '.bashrc')
+        if Path(shell_command).name.lower() == 'bash' and bashrc_location.is_file():
+            # set PS1 after bash execution if shell program is bash and if bashrc is found
+            temp_rc = mkstemp()
+            with open(temp_rc[1], 'w') as rc_file:
```

Comment:
I think there is no reason for using `mkstemp()` such a low-level interface.

Suggestion:
```
with tempfile.NamedTemporaryFile('w', delete=False) as rc_file:
    ...
````
and we can use `rc_file.name` instead of `temp_rc[1]`.
In any case, unlink is necessary at the end.

Reply:
I used NamedTemporaryFile instead, and it is a bit cleaner.